### PR TITLE
🆕 Adding new operator and updating documentation📈

### PR DESCRIPTION
**Pull request by Asish-dot-Rout**
I have changed the following things:
1. Added Operator definition and types of operator
2. Added Unary operator
3. Updated the documentation with proper indentation

### CODE REVIEW

File: Operator.java [hunk 1]
```diff
@@ -1,10 +1,68 @@
...
+     * 
+     * 
+     */
+    int m4 = 20;
+    int z = ++m4;//First m4 will increase by one and then the value of m4 will assign to z(pre increment)
+    int z1 = m4++;//First the value of m4 will assign to z and then m4 will increase by one(post increment)
+
+    System.out.println(z);
+    System.out.println(z1);
+
```
Comment (Dev-Soumyaranjan) [inline Operator.java:41]: Comment out the output of above two print statements

**Review by Dev-Soumyaranjan — CHANGES_REQUESTED** [review]
Thanks👍 for adding new operator and updating the documentation✨. It looks good🎉
Please go through my comments to make this PR eligible for merging😊

File: Operator.java (hunk 1 above)
Comment (Asish-dot-Rout) [reply Operator.java:41]: Ok fine👍

**Review by Dev-Soumyaranjan — APPROVED** [review]
Good🎉

**Pull request merged**